vcpkg.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "colmap",
+    "description": "COLMAP is a general-purpose Structure-from-Motion (SfM) and Multi-View Stereo (MVS) pipeline with a graphical and command-line interface. It offers a wide range of features for reconstruction of ordered and unordered image collections. The software is licensed under the new BSD license.",
+    "homepage": "https://colmap.github.io/",
+    "license": "BSD-3-Clause",
+    "supports": "(linux | (windows & !static) | osx) & (x86 | x64 | arm64)",
+    "dependencies": [
+        "boost-algorithm",
+        "boost-filesystem",
+        "boost-graph",
+        "boost-heap",
+        "boost-program-options",
+        "boost-property-map",
+        "boost-property-tree",
+        {
+            "name": "ceres",
+            "features": [
+                "lapack",
+                "suitesparse"
+            ]
+        },
+        "cgal",
+        "eigen3",
+        "flann",
+        "freeimage",
+        "gflags",
+        "glew",
```

Comment:
Shouldn't this be required only when the GUI is enabled? We don't build it in the pycolmap CI.

Reply:
Good catch. Thx.